mmdet/models/necks/bfp.py

```diff
@@ -0,0 +1,163 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import xavier_init
+
+from ..plugins import NonLocalBlock2D
+from ..utils import ConvModule
+from ..registry import NECKS
+
+
+@NECKS.register_module
+class BFP(nn.Module):
+    """BFP (Balanced Feature Pyrmamids)
+
+    arXiv: https://arxiv.org/pdf/1904.02701.pdf (CVPR 2019)
+    """
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 num_outs,
+                 start_level=0,
+                 end_level=-1,
+                 add_extra_convs=False,
+                 extra_convs_on_inputs=True,
+                 relu_before_extra_convs=False,
+                 conv_cfg=None,
+                 norm_cfg=None,
+                 refine_level=2,
+                 refine_type=None,
+                 activation=None):
```

Comment:
The refine module is before an element-wise sum, so `activation` may not be useful since it is always preferred to be `None`.